packages/eslint-plugin/typings/eslint-keywords.d.ts

```diff
@@ -0,0 +1,69 @@
+declare type Keyword =
```

Comment:
It would be nice to generate this type from the `KEYWORDS` string array in `astUtils.ts`, just like it done in https://stackoverflow.com/a/54061487/1541141
Unfortunately, I am not able to declare a value in a `d.ts` file.
Any way to make this nice?

Reply:
This file is only needed if you're importing from `eslint-keywords` - since you've now pulled it into `astUtils`, you don't need this :)

Reply:
The issue here is that I need to know all the possible values of `Keyword` in order to build my `Options` type. However, if I import the list of keywords from `eslint`, the list is only known at runtime.

Reply:
Yeah this is an interesting one.

I think for now the best way to handle this is to type it as `string`, and look at where it breaks - I've just realised that we're going to have to be very careful about this rule anyway due to how the base rule works.

I'll put the details in another comment on the more-relevant lines :)